modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -27,12 +27,52 @@ kubectl describe nodes
 
-On a worker node, Kubernetes and Redpanda processes are running at the same time, including the Seastar subsystem that is built into the Redpanda binary. Each of these processes consumes memory. You can configure the memory resources that are allocated to these processes.
+When deploying Redpanda, you must reserve sufficient memory for both Redpanda and other system processes. Redpanda uses the Seastar framework to manage memory through two important flags. In Kubernetes, the values of these flags are usually set for you, depending on how you configure the Redpanda CRD or the Helm chart.
 
-By default, the Helm chart allocates 80% of the configured memory in `resources.memory.container` to Redpanda, with the remaining reserved for overhead such as the Seastar subsystem and other container processes.
-Redpanda Data recommends this default setting.
+- **`--memory`**: When set, explicitly defines the Redpanda heap size.
+- **`--reserve-memory`**: When set, reserves a specific amount of memory for system overhead. If not set, a reserve is automatically calculated.
+
+.Learn more about these Seastar flags
+[%collapsible]
+====
+
+[cols="1,1,2"]
+|===
+|                        | **`--memory` Not set**
+|                       **`--memory` Set (M)**
+
+| **`--reserve-memory` Not set**  
+| Heap size = available memory - calculated reserve
+| Heap size = exactly M (if M plus calculated reserve ≤ available memory). Otherwise, startup fails
+
+| **`--reserve-memory` set \(R)**
+| Heap size = available memory - R
+| Heap size = exactly M (if M + R ≤ available memory). Otherwise, startup fails
+|===
+
+Definitions and behavior:
+
+- **Available memory**: The memory remaining after subtracting system requirements, such as `/proc/sys/vm/min_free_kbytes`, from the total or cgroup-limited memory.
+- **Calculated reserve**: The greater of 1.5 GiB or 7% of _available memory_ used when `--reserve-memory` is not explicitly set.
+
+====
+
+**Legacy (default) behavior**: By default, the Helm chart and the Redpanda CRD define memory requirements using the `resources.memory.container` configuration, which allocates:
+
+- 80% of the container memory to `--memory`
+- 20% to `--reserve-memory`
+
+CAUTION: This default legacy behavior is for backward compatibility only. **Do not use it in production.**
+
+**Production recommendation**: Use `resources.requests.memory` for production deployments to ensure predictable memory allocation by letting Kubernetes schedule and enforce resource limits. When you set a value for `resources.requests.memory`:
+
+- Kubernetes allocates 90% of the requested memory to the Redpanda heap (using `--memory`).
++
+The remaining 10% of the requested memory is reserved for container overhead. This allocation ensures that system processes and runtime requirements have sufficient memory while providing a predictable, dedicated heap for Redpanda.
+- The `--reserve-memory` value is fixed at 0 because Kubernetes already manages container overhead.
```

Comment:
Still picking on container overhead here. Though I think it would be good to distinguish the intended purpose of the `--reserve-memory` flag and the 10% overhead we carve out.

```suggestion
- The `--reserve-memory` value is fixed at 0 because Kubelet already manages system level memory reservations.
```